mvp.adoc

```diff
@@ -188,47 +189,26 @@ a| TBD
 | Sizing Source
 
 | Master 0
-| 4
-| 16
+| 8
+| 32
 | 120
-| m4.xlarge
+| m4.2xlarge
 | https://docs.openshift.com/container-platform/4.3/scalability_and_performance/recommended-host-practices.html#master-node-sizing_[OCP 4 Docs - Master Node Sizing]
 
 | Master 1
-| 4
-| 16
+| 8
+| 32
 | 120
-| m4.xlarge
+| m4.2xlarge
 | https://docs.openshift.com/container-platform/4.3/scalability_and_performance/recommended-host-practices.html#master-node-sizing_[OCP 4 Docs - Master Node Sizing]
 
 | Master 2
-| 4
-| 16
+| 8
+| 32
 | 120
-| m4.xlarge
+| m4.2xlarge
 | https://docs.openshift.com/container-platform/4.3/scalability_and_performance/recommended-host-practices.html#master-node-sizing_[OCP 4 Docs - Master Node Sizing]
 
-| Infra 0
-| 4
-| 16
-| 120
-| m4.xlarge
-| WAG based on OCP 3 recomendations since infra nodes for OCP 4 are just starting to become a thing again.
-
-| Infra 1
-| 4
-| 16
-| 120
-| m4.xlarge
-| WAG based on OCP 3 recomendations since infra nodes for OCP 4 are just starting to become a thing again.
-
-| Infra 2
-| 4
-| 16
-| 120
-| m4.xlarge
-| WAG based on OCP 3 recomendations since infra nodes for OCP 4 are just starting to become a thing again.
-
```

Comment:
@ckupe are we striking the infra nodes?

Reply:
@kfrankli yes, CEE does not use infra nodes in their cluster deployments. OCP4 cluster deployment is mainly ignition bootstrap node -> 3 masters then the machineset API is leveraged to compose the workers dynamically with declared machinesets posted to the API. You can technically pull out the routing and management components into infra nodes just like in OCP3, but many people have said not to do that.

Reply:
Instead of using a statically composed elastic load balancer at the edge with specific infra routers as targets, they create a network based ELB with target groups, then those target groups have port criteria added and joined to the ELBs. So the routing for the apps/data plane is more "dynamic" rather than statically configured through traditional router nodes (traditionally the infra nodes).